apps/mull-ui/src/app/pages/profile/edit-profile/edit-profile.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import EditProfile from './edit-profile';
+
+describe('EditProfile', () => {
```

Comment:
⚠ Coverage is pretty low on this component
![image](https://user-images.githubusercontent.com/23544999/107155695-d27e0200-6947-11eb-852c-0f0cae4cd156.png)

Add some tests that click on the button and test the form code